src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -178,7 +180,7 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
         <ReactResizeDetector handleWidth={true} handleHeight={true} skipOnMount={true} onResize={this.onResize} />
         <EmptyGraphLayout
           elements={this.props.elements}
-          namespace={store.getState().namespaces.activeNamespace.name}
+          namespace={activeNamespacesAsStringSelector(store.getState())}
```

Comment:
I think the property should go from 'namespace' to 'namespaces' and I think the conversion to a comma-separated-string value should be done only where it's needed (i.e. in EmptyGraphLayout).   Also, on a separate topic, we will probably want to stop accessing the store directly.

Reply:
Yes, not in this PR.  Actually, I may do some of that today in a separate PR, as a follow-up to Edgar's comments on the 1880 PR yesterday.